afids_utils/ext/json.py

```diff
@@ -0,0 +1,154 @@
+"""Methods for handling .json files associated with AFIDS"""
+from __future__ import annotations
+
+import json
+from importlib import resources
+from os import PathLike
+
+from afids_utils.afids import AfidPosition, AfidSet
+from afids_utils.exceptions import InvalidFileError
+
+
+class ControlPoint:
```

Comment:
I don't think this gets used anywhere (looks like it may have been intended to replace the template approach?)

Reply:
Now what we could do if you want is make this a `TypedDict` and use it as a type hint in `_get_afids`

Reply:
Ahh you're right, I started playing around with this with the intention of making this a `TypedDict` and then forgot about it when I went back to it.